### ISSUE TITLE
Ansible with winrm 

### ISSUE THREAD

**Issue by alexray92** [rundeck-plugins/ansible-plugin#152]
I compiled the latest of the master branch and deployed it to my rundeck server - I have two hosts configured in a Windows group, which seems to work in ansible: I can do -m ping and -m setup and it gathers facts fine.  But when I try to set up a project in rundeck to use those Windows nodes, it fails here:

`
[WARNING]: Unable to set correct type for configuration entry:
DEFAULT_LOCAL_TMP

PLAY [all] *********************************************************************

TASK [Gathering Facts] *********************************************************
ok: [winhost1]

TASK [file] ********************************************************************
fatal: [winhost1]: UNREACHABLE! => {"changed": false, "msg": "Authentication or permission failure. In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote temp path in ansible.cfg to a path rooted in \"/tmp\". Failed command was: ( umask 77 && mkdir -p \"` echo /var/lib/rundeck/.ansible/tmp/ansible-tmp-1507304672.03-229903121842383 `\" && echo ansible-tmp-1507304672.03-229903121842383=\"` echo /var/lib/rundeck/.ansible/tmp/ansible-tmp-1507304672.03-229903121842383 `\" ), exited with result 1", "unreachable": true}

NO MORE HOSTS LEFT *************************************************************
        to retry, use: --limit @/tmp/rundeck/ansible-hosts4326398628209340940/gather-hosts.retry

PLAY RECAP *********************************************************************
winhost1   : ok=1    changed=0    unreachable=1    failed=0
` 

Thinking that the problem was due to the setting of remote_tmp, I changed remote_tmp in ansible.cfg to be the same as what is reported in ansible -m setup for winhost1 -> ansible_facts -> ansible_env -> "TMP".  After restarting rundeck, the node information populated and I was able to target the nodes in a job executing the remote command "ipconfig".

However, the job failed with this message:

`
Failed: ParseArgumentsError: Failed to read the become password for storage path: /etc/rundeck/keys/xxxxxxx: Unauthorized access

09:16:44 | localhost |   | Execution failed: 15125 in project ScheduledTasks: [Workflow result: , step failures: {1=Dispatch failed on 1 nodes: [winhost1: ParseArgumentsError: Failed to read the become password for storage path: /etc/rundeck/keys/xxxxxxxxx: Unauthorized access + {dataContext=MultiDataContextImpl(map={}, base=null)} ]}, Node failures: {winhost1=[ParseArgumentsError: Failed to read the become password for storage path: /etc/rundeck/keys/xxxxxxxx: Unauthorized access + {dataContext=MultiDataContextImpl(map={}, base=null)} ]}, status: failed]
`

Which seems to indicate that it's attempting SSH authentication instead of WinRM/powershell as is configured in the default node executor (this plugin) for the project.  I removed all SSH configuration from the node executor, after which ipconfig successfully ran, though still with this warning:

`
[WARNING]: Unable to set correct type for configuration entry:

09:20:10 |   |   | DEFAULT_LOCAL_TMP
`

tl;dr I can get Windows hosts over WinRM in ansible to populate in Rundeck, and get commands to run against them, but only if I:

1. Change remote_tmp in ansible.cfg to be the same as what is reported in ansible -m setup for winhost1 -> ansible_facts -> ansible_env -> "TMP"

2. Do not configure any SSH settings in the ansible plugin for the project

Which means I can only execute against Windows hosts using ansible, not linux hosts.  

I realize Windows support is pretty early, but I feel like this could be fixed by just checking if "ansible_os_family" is "Windows", and if so, skipping SSH and changing remote_tmp.

**Comment by mcassaniti** [comment]
Is this issue still occurring for you on the latest version, and can you confirm that running Ansible directly works as expected (i.e.: outside of Rundeck)?

**Comment by mcassaniti** [comment]
Closing since there's been no further update.
